apps/src/templates/studioHomepages/ContentContainer.jsx

```diff
@@ -111,7 +111,6 @@ const ContentContainer= React.createClass({
           return (
             <div key={index}>
               {child}
-              {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
I moved the spacer into the mapping of courses in the RecentCourses component because I wanted to put components of various sizes into the content container.